package/content/example.php

```diff
@@ -0,0 +1,140 @@
+﻿<?php
```

Comment:
Could we reduce or remove the provided demo files?
Could we also share them between the Docker images and the packages?

@ginifizz will take a look to the graphic design part.